### DISCUSSION TITLE
Plugin are not working fine for "metrics-by-endpoint"

### DISCUSSION THREAD

**Discussion by ld0906** [artilleryio/artillery#1249] [Artillery]
Hi there,

I am not sure if someone met the plugin issue as below, I tried to uninstall and then install plugin, not working fine. Also tried install it globally or project base, also not working.
 Thanks.
```
WARNING: Could not load plugin: metrics-by-endpoint warn 

WARNING: Plugin metrics-by-endpoint specified but module artillery-plugin-metrics-by-endpoint could not be found (MODULE_NOT_FOUND) warn 
```

information for artillery:

artillery version: 2.0.0-6 
OS: linux-x64 
Node version: v12.22.7

**Comment by hassy** [comment]
hi @ld0906 👋  this usually happens when Artillery and a plugin are not **both** installed either globally or locally. Can you try installing both globally for example, with:

```sh
npm install -g artillery artillery-plugin-publish-metrics
```
